### PR TITLE
RavenDB-19342: Fix for baseline insertion error

### PR DESCRIPTION
**Pull request by redknightlois**

### CODE REVIEW

**Comment by redknightlois** [comment]
cc @ayende

File: test/FastTests/Voron/Sets/SetTests.cs [hunk 1]
```diff
@@ -245,6 +245,48 @@ public void CanAddPredictableOffsets_Large()
             }
         }
 
+        [Fact]
+        public void CanAddHugeOffsets_Large()
+        {
```
Comment (redknightlois) [inline test/FastTests/Voron/Sets/SetTests.cs:250]: This is the error from the licenses database.

File: test/FastTests/Voron/Sets/SetTests.cs [hunk 2]
```diff
@@ -451,5 +493,74 @@ static void Shuffle(int[] list, Random rng)
                 }
             }
         }
+
+        [RavenTheory(RavenTestCategory.Voron)]
+        [InlineData(1279, 51251, true)]
+        public void CanDeleteAndInsertInBulk(int seed, int size, int maxValue = 1000000, bool includeDuplicates = false)
+        {
```
Comment (redknightlois) [inline test/FastTests/Voron/Sets/SetTests.cs]: This is another one I found by chance

**Comment by redknightlois** [comment]
Already reviewed with @ayende on conference call.

**Review by ayende — APPROVED** [review]

**Pull request merged**